main.py

```diff
@@ -1,10 +1,56 @@
-import init_django_orm  # noqa: F401
+from typing import Callable, Union
 
+import init_django_orm  # noqa: F401
+import json
 from db.models import Race, Skill, Player, Guild
 
 
+def race_creating(race_dict: dict) -> Callable:
+    if not Race.objects.filter(name=race_dict["race"]["name"]).exists():
+        return Race.objects.create(
+            name=race_dict["race"]["name"],
+            description=race_dict["race"]["description"]
+        )
+    else:
+        return Race.objects.get(name=race_dict["race"]["name"])
```

Comment:
You can use get_or_create
The same with guild
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create

Reply:
Also as I see you always do: race_dict["race"] rewrite it. race_dict == gamer_values["race"]

Reply:
The same comments to guild